dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -752,6 +759,24 @@ struct RegionUsedByArgs {
     region_id: Vec<Uuid>,
 }
 
+#[derive(Debug, Args, Clone)]
+struct DryRunRegionAllocationArgs {
```

Comment:
It would be nice to document these arguments.

e.g.:
- snapshot_id - what does this do? (I know this implementation relies on  `region_allocation.rs`, but it would be nice to show up in the CLI help message)
- "size" is the size of the "virtual disk"
- "num_regions_required" is usually 3

Reply:
:+1: done in 427ef4442